server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/CVRExportUpload.java

```diff
@@ -182,25 +184,25 @@ private void handleUpload(final Request the_request,
       the_info.my_ok = false;
     }
   }
-  
+
   /**
    * Parses an uploaded CVR export and attempts to persist it to the database.
    * 
    * @param the_info The upload information to use and update.
    */
   // the CSV parser can throw arbitrary runtime exceptions, which we must catch
   @SuppressWarnings({"PMD.AvoidCatchingGenericException", "PMD.AvoidCatchingNPE"})
-  private void parseAndPersistFile(final UploadInformation the_info) {
+  private void parseAndPersistFile(final Response the_response,
```

Comment:
Hmm, it seems like `parseAndPersistFile()` shouldn't have to know anything about `the_response`.